sources/packages/web/src/composables/useFormioDropdownLoader.ts

```diff
@@ -204,17 +204,14 @@ export function useFormioDropdownLoader() {
     );
   };
 
-  const loadInstitutionName = async (
-    form: FormIOForm,
-    dropdownName: string,
-  ) => {
-    // Find the dropdown to be populated with the locations.
-    const dropdown = formioUtils.getFirstComponent(form, dropdownName);
-    return loadDropdown(
-      form,
-      dropdown,
-      InstitutionService.shared.getInstitutionNameOptions(),
-    );
+  const loadInstitutionNames = async (form: FormIOForm, apiKey: string) => {
```

Comment:
We do not use `apiKey` anywhere else as far as I could check. Can you please change it to `componentKey` following the same from the method `setComponentValue`.